run/GCHP/ExtData.rc.templates/ExtData.rc.carbon

```diff
@@ -135,63 +135,71 @@ XLAIMULTI cm2_cm-2 N Y %y4-%m2-%d2T00:00:00 none none XLAIMULTI ./HcoDir/Yuan_XL
 
 #==============================================================================
 # %%%%% CH4 emissions %%%%%
+#
+# Comment all entries in this section when
+# "--> USE_CH4_DATA : false" is in HEMCO_Config.rc.
```

Comment:
Why do we need to comment these lines if USE_CH4_DATA is false? If the variable names are not used in other inventories we do not need to comment them out. Indeed, commenting them out will actually cause problems for people unless they are automatically uncomments when updating HEMCO_Config.rc.

Reply:
Comments have now been cleaned up in commit https://github.com/geoschem/geos-chem/pull/1992/commits/e837aba3ec9d112b4ab4cc501707e621e43db042.